ct-app/core/model/peer.py

```diff
@@ -122,9 +123,13 @@ async def message_delay(self) -> float:
         count = await self.yearly_message_count.get()
 
         if count is not None and count > 0:
-            return (365 * 24 * 60 * 60) / count
+            value = (365 * 24 * 60 * 60) / count
```

Comment:
Magic variable that breaks on the skip year:
```suggestion
 SECONDS_IN_A_NON_LEAP_YEAR = 365 * 24 * 60 * 60

...

            value = SECONDS_IN_A_NON_LEAP_YEAR / count
```